### PR TITLE
Remove deprecated iGEMM xdlops solver and kernels

### PR DESCRIPTION
**Pull request by asroy**
This PR remove deprecated iGEMM xdlops solver and kernels. 

These  forward solvers are removed:
```
ConvHipImplicitGemmV4R4GenXdlopsFwdFp32
ConvHipImplicitGemmV4R4GenFwdXdlops
```

These backward-weight solvers are removed
```
ConvHipImplicitGemmV4R4GenXdlopsWrWFp32
ConvHipImplicitGemmV4R4GenWrWXdlops
```

### CODE REVIEW

File: src/kernels/composable_kernel/include/kernel_algorithm/gridwise_convolution_backward_data_implicit_gemm_v4r1_xdlops_gnchw_gkcyx_gnkhw.hpp [hunk 1]
```diff
@@ -26,18 +27,17 @@ template <index_t GridSize,
...
-          typename GemmABlockCopyThreadClusterLengths_GemmG_GemmK_GemmM,
+          typename GemmABlockCopyThreadSliceLengths_GemmG_GemmK_GemmM_GemmKPACK,
+          typename GemmABlockCopyThreadClusterLengths_GemmG_GemmK_GemmM_GemmKPACK,
           index_t GemmABlockCopySrcDataPerRead_GemmM,
-          index_t GemmABlockCopyDstDataPerWrite_GemmM,
-          typename GemmBBlockCopyThreadSliceLengths_GemmG_GemmK_GemmN,
-          typename GemmBBlockCopyThreadClusterLengths_GemmG_GemmK_GemmN,
+          index_t GemmABlockCopyDstDataPerWrite_GemmKPACK,
+          typename GemmBBlockCopyThreadSliceLengths_GemmG_GemmK_GemmN_GemmKPACK,
+          typename GemmBBlockCopyThreadClusterLengths_GemmG_GemmK_GemmN_GemmKPACK,
```
Comment (JehandadKhan) [inline src/kernels/composable_kernel/include/kernel_algorithm/gridwise_convolution_backward_data_implicit_gemm_v4r1_xdlops_gnchw_gkcyx_gnkhw.hpp:38]: why are these changes required to remove the solvers ?

**Review by JehandadKhan — CHANGES_REQUESTED** [review]
Can you please confirm if the invoker related code has been removed as well ?

File: src/kernels/composable_kernel/include/kernel_algorithm/gridwise_convolution_backward_data_implicit_gemm_v4r1_xdlops_gnchw_gkcyx_gnkhw.hpp (hunk 1 above)
Comment (asroy) [reply src/kernels/composable_kernel/include/kernel_algorithm/gridwise_convolution_backward_data_implicit_gemm_v4r1_xdlops_gnchw_gkcyx_gnkhw.hpp:38]: I removed some file and change some file name. Git diff thought I modified files

**Comment by asroy** [comment]
> Can you please confirm if the invoker related code has been removed as well ?

Just removed

**Review by JehandadKhan — APPROVED** [review]
